packages/cli/src/commands/destroy/page/__tests__/page.test.js

```diff
@@ -0,0 +1,53 @@
+global.__dirname = __dirname
+jest.mock('fs')
+jest.mock('src/lib', () => {
+  return {
+    ...require.requireActual('src/lib'),
+    generateTemplate: () => '',
+  }
+})
+
+import fs from 'fs'
+
+import 'src/lib/test'
+import { getPaths } from 'src/lib'
+
+import { files } from '../../../generate/page/page'
+import { tasks } from '../page'
+
+beforeEach(() => {
+  fs.__setMockFiles({
+    ...files({ name: 'About' }),
+    [getPaths().web.routes]: [
+      '<Routes>',
+      '  <Route path="/about" page={AboutPage} name="about" />',
+      '  <Route path="/" page={HomePage} name="home" />',
+      '  <Route notfound page={NotFoundPage} />',
+      '</Routes>',
+    ].join('\n'),
+  })
+})
+
+test('destroys a page and route', async () => {
```

Comment:
How gross would it be to split this into two tests? Can you do separate "destroy page" and a "destroy routes" tests? And just mock the page stuff in the page test and the routes stuff in the route test?

Reply:
Will the destroyers just keep going if something they intend to destroy doesn't exist any more? Specifically, if you only mock the page, will the route destroyer fail, or vice versa?